picoharpApp/src/picopeaks.c

```diff
@@ -3,12 +3,11 @@
 #include <stdlib.h>
 #include <math.h>
 #include <string.h>
-#include <unistd.h>
+//#include <unistd.h>
```

Comment:
Hmm.  What's this about?  Why remove this `#include`, and why leave it in as a comment?  Either remove it with an explanation or leave it alone.

Reply:
I could not find this header file.